# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/lzdoom/hotfix.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- gzdoom-3.88a/libraries/music_common/fileio.h.vanilla	2024-11-21 15:17:29.173333324 +0100
+++ gzdoom-3.88a/libraries/music_common/fileio.h	2024-11-21 15:17:39.499999997 +0100
@@ -24,6 +24,7 @@
 
 #pragma once
 #include <stdio.h>
+#include <inttypes.h>
 #include <string.h>
 #include <vector>
 #include <string>
--- gzdoom-3.88a/libraries/zmusic/zmusic/zmusic_internal.h.vanilla	2024-11-21 15:19:53.133333327 +0100
+++ gzdoom-3.88a/libraries/zmusic/zmusic/zmusic_internal.h	2024-11-21 15:20:01.869999997 +0100
@@ -39,6 +39,8 @@
 #include "zmusic.h"
 #include "../../music_common/fileio.h"
 
+#include <stdexcept>
+
 void SetError(const char *text);
 
 struct CustomFileReader : public MusicIO::FileInterface
